### DISCUSSION TITLE
Feature request: methods for "open" and "close"

### DISCUSSION THREAD

**Discussion by bjsintay** [snapappointments/bootstrap-select#2636] [Ideas]
Seems like you can do what you need with the various methods except open and close the dropdown explicitly. Certainly, the "toggle" method works for this, but there are times when you need to close the dropdown no matter what the current state is, and that is where toggle runs short. It would be great to have "open" and "close" methods in addition to "toggle".

Thanks for your hard work on this project and consideration of this idea!

**Comment by tf-rshearer** [comment]
https://github.com/snapappointments/bootstrap-select/issues/1893 Its been a while but this is apart of a [beta release](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.14.0-beta)
